.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: python
 python:
- - "2.6"
```

Comment:
There is (at least) one downstream project which maintains 2.6 compatibility which I'd rather not break.

Reply:
Python 2.6 is officially not supported. Are you sure this is a good thing to spend time on?